### PR TITLE
Feature/blake yarbrough update

### PR DESCRIPTION
**Pull request by LanguidSquid**
Merging BlakeYarbroughUpdate
Fixes:
-Failing unit test of Message of the Day GET service
Adds:
-Message of the Day POST service
-Unit test of Message of the Day POST service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,4 +40,11 @@ AWS or Heroku, or whatever you're comfortable with will be an "A+"
   * The official Spring Boot "hello world" example is a great starting point.
 * Still need help?
   * Further hints are available, Feel free to ask questions here.  Edit this file in your branch by adding to the 
-questions section, push it, and we will update the file with answers. 
\ No newline at end of file
+questions section, push it, and we will update the file with answers. 
+
+### Blake Yarbrough Progress
```
Comment (dnellis74) [inline README.md]: Thanks for updating the readme.md!

File: src/main/java/com/charter/enterprise/motd/MotdController.java [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.charter.enterprise.motd.beans.MessageOfTheDayPostRequest;
+import org.springframework.web.bind.annotation.*;
 
 @RequestMapping("/")
 @RestController
 public class MotdController {
 
+    //bad idea
```
Comment (dnellis74) [inline src/main/java/com/charter/enterprise/motd/MotdController.java]: Your comment is spot on, but this is indeed an okay solution, give the amount of time we expected you to spend.

File: src/test/java/com/charter/enterprise/motd/MotdControllerTest.java [hunk 3]
```diff
@@ -26,4 +29,15 @@ public void getMotd() throws Exception {
                 .andExpect(status().isOk())
                 .andExpect(content().string(equalTo("Goodbye world!")));
     }
+
+    @Test
+    public void postMotd() throws Exception {
```
Comment (dnellis74) [inline src/test/java/com/charter/enterprise/motd/MotdControllerTest.java]: Great job adding the test too.

**Comment by david-nellis-charter** [comment]
Thanks Blake.   I've saved your branch in a private repo.

**Pull request closed without merging**